packages/sdk-cli/src/models/HostConnections.ts

```diff
@@ -12,9 +15,37 @@ export class HostConnection {
     public host: RemoteHost,
   ) {}
 
+  static getDumpStreamTap() {
+    const dumpFileHandle = fs.openSync(process.env.FITBIT_DEVBRIDGE_DUMP_PATH as string, 'w');
```

Comment:
isn't opening a file every message, expensive? Just pointing it out, don't (necessarily) forsee any issues here.